super_dash_counter/analysis_options.yaml

```diff
@@ -0,0 +1,4 @@
+include: package:flutter_lints/flutter.yaml
```

Comment:
Also might want to use very_good_analysis to be consistent with the other shorebird projects 🤷

Reply:
Oh, I checked on the flavors one and I saw just the flutter lints, so I thought that was the standard for samples. I will def change